osu.Game/Overlays/Profile/UserGraph.cs

```diff
@@ -268,7 +265,12 @@ private void load(OsuColour colours)
                 background.Colour = colours.Gray1;
             }
 
-            public abstract void SetContent(object content);
+            public void SetContent(UserGraphTooltipContent content)
+            {
+                Label.Text = content.Name;
+                Counter.Text = content.Count;
+                BottomText.Text = content.Time;
+            }
```

Comment:
This is broken now that `UserGraphTooltipContent` is a struct. See this screenshot taken in `TestSceneRankGraph`:

![2021-08-31-195723_545x336_scrot](https://user-images.githubusercontent.com/20418176/131552786-615ab1ed-bd03-4392-83ba-337b786899d3.png)

There should be no tooltip there. The derived classes had `is` typechecks that doubled as null-checks, which are broken now that this is a struct.

(Also the gains of making `UserGraphTooltipContent` a struct are disputable in the first place, since the whole tooltip API is still `object`-based, so the struct is most likely getting boxed somewhere along the way anyway.)

Reply:
Right, didn't realize that. I'll go for `class` then.